scripts/run_e2e_tests.sh

```diff
@@ -240,14 +240,14 @@ done
 # TODO(bhenning): Figure out if this is a bug with protractor.
 if [ "$RUN_ON_BROWSERSTACK" == "False" ]; then
   if [ "$SHARDING" = "false" ] || [ "$SHARD_INSTANCES" = "1" ]; then
-    node_modules/protractor/bin/protractor core/tests/protractor.conf.js --suite "$SUITE" --params.devMode="$DEV_MODE"
+    node_modules/.bin/percy exec -- node_modules/protractor/bin/protractor core/tests/protractor.conf.js --suite "$SUITE" --params.devMode="$DEV_MODE"
```

Comment:
Just checking a few things:

1. Do e2e test failures error logs still show up correctly, with this change?

2. What happens when the e2e tests are run locally? Do they still run fine? What about the percy statements?